src/libraries/System.IO.FileSystem/src/System/IO/Enumeration/FileSystemEnumerator.Win32.cs

```diff
@@ -55,7 +55,7 @@ private unsafe bool GetData()
             }
         }
 
-        private unsafe IntPtr CreateRelativeDirectoryHandle(ReadOnlySpan<char> relativePath, string fullPath)
+        private unsafe IntPtr CreateRelativeDirectoryHandle(ReadOnlySpan<char> relativePath, string? fullPath)
```

Comment:
Can `fullPath` be nullable? It doesn't seem like the usages that we have will pass down null.

Reply:
for this method null allowed, but the only calling method of this not passing nullable so this is not necessary to be nullable